modules/insights-operator-document-attributes.adoc

```diff
@@ -0,0 +1,10 @@
+////
+This is a custom set of attributes for the Insights Operator related documentation. 
+////
+
+:insights-title: Insights Advisor
+
+:link-cloud-redhat-com: link:https://cloud.redhat.com/[cloud.redhat.com]
```

Comment:
@vikram-redhat, do you have thoughts on placing these attributes in a separate file instead of using them more widely in the collection?